### PR TITLE
adding strings for copy and fixing a typo

### PR DESCRIPTION
**Pull request by deilann**
le strings for the copy

and there was a random "1" hanging out where it didn't belong.

### CODE REVIEW

**Comment by deilann** [comment]
This is the typo: https://github.com/HabitRPG/habitrpg/issues/2548

**Comment by marcelosizer** [comment]
@lefnire this will conflict with #119 as main.json got removed. @deilann can you add it to one of the files in #119 as main.json will be deleted?

**Comment by deilann** [comment]
Will do.

**Comment by marcelosizer** [comment]
Thanks! =]

**Comment by paglias** [comment]
@Sinza- don't worry about that, I'll solve the conflicts when I'll merge your pr (tomorrow, I hope)

**Comment by marcelosizer** [comment]
Ok then!

**Comment by lefnire** [comment]
Sorry @Sinza-, my bad!

File: locales/en/main.json [hunk 1]
```diff
@@ -7,7 +7,20 @@
 
 "_commentfrontpage":"HABITRPG FRONT PAGE",
   "titleFront" : "HabitRPG | Gamify Your Life",
-  "synopsis" : "A habit building program which treats your life like a Role Playing Game. Level up as you succeed, lose HP as you fail, earn money to buy weapons and armor.",
+  "tagline" : "The only free habit building app that treats your life like a game.",
```
Comment (lefnire) [inline locales/en/main.json:10]: @deilann sorry, I just realized I commented on the wrong line a way back when. It's this one: 

> The only free habit building app that treats your life like a game.

That's what I meant, there's others out there that fit this bill. Thoughts?

Comment (deilann) [inline locales/en/main.json:10]: That's a good point.

I would definitely classify the others as "habit building apps," even
though they're niche. Switch habit building app for productivity manager,
I'd say, then tweak habit building app into somewhere else. I'll have to
rebalance for that.

**Pull request merged**